NEWS.md

```diff
@@ -1,3 +1,8 @@
+# shiny.fluent 0.2.1
+
+* Ensure `Calendar.shinyInput` and `DatePicker.shinyInput` with `value = NULL` to behave
+  consistently with the official behavior. [#118](https://github.com/Appsilon/shiny.fluent/pull/118)
+
```

Comment:
@jakubsob since this PR will introduce breaking changes (return values of `Calendar` & `DatePicker` changed), is it better we increment minor version eg. 0.3.0?

Reply:
The package is still not on CRAN and the version is `0.[...]` so breaking changes are expected to occur. Let's increment the version to 0.2.2